### PR TITLE
Ei homog branch: Adding on election algebra function and rd file

### PR DESCRIPTION
**Pull request by lorenc5**
Will add test() as needed once I get some review!

### CODE REVIEW

File: R/elect_algebra.R [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+elect_algebra <- function(to, 
+                          cvap, 
+                          c1_ei_res, 
+                          c2_ei_res, 
+                          cand_names) {
+    
+    df <- data.frame(to, cvap, stringsAsFactors = F)
+    
+    # Set Column Output Names #
+    cnames =c("White_Vote", "NonWhite_Vote")
```
Comment (pssachdeva) [inline R/elect_algebra.R:84]: nitpick: space around equals size

File: R/elect_algebra.R [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    
+    # 1 is whites, 2 is non-whites
+    votes <- round(apply(df, 1, prod), 0) 
+    votes <- c(votes, sum(votes))
+    
+    # White Vote (for 2 candidates)
+    wv <- round( votes[1] * c1_ei_res, 0)
```
Comment (pssachdeva) [inline R/elect_algebra.R:91]: nitpick: space after parenthesis

File: R/elect_algebra.R [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    nwv <- round( votes[2] * c2_ei_res, 0)
+    out <-  data.frame (wv, nwv, total = wv + nwv, stringsAsFactors = F) 
+    out <- rbind(out, votes)
+    
+    # Collect column/row names 
+    colnames(out) <- c(cnames, "Total")
+    row.names(out) <- c(cand_names, "Total")
+    
+    # Add back on Percentages #
+    out$Pct_White <- c(round( out$White_Vote[1] / out$White_Vote[3],3),
```
Comment (pssachdeva) [inline R/elect_algebra.R:103]: nitpick: spaces after parentheses

File: R/elect_algebra.R [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    # 1 is whites, 2 is non-whites
+    votes <- round(apply(df, 1, prod), 0) 
+    votes <- c(votes, sum(votes))
+    
+    # White Vote (for 2 candidates)
+    wv <- round( votes[1] * c1_ei_res, 0)
+    
+    # Non-White Vote (for 2 candidates)
+    nwv <- round( votes[2] * c2_ei_res, 0)
+    out <-  data.frame (wv, nwv, total = wv + nwv, stringsAsFactors = F)
```
Comment (pssachdeva) [inline R/elect_algebra.R:95]: nitpick: spaces before parentheses

File: R/elect_algebra.R [hunk 5]
```diff
@@ -0,0 +1,112 @@
+#' Election Algebra for 2x2 Case
+#'
+#' Creates data.frame() table of algebraically defined white/non-white preferences for candidates. Typically used when analyst has high confidence in white turnout and voting behavior but needs to deduce minority voting behavior when only CVAP available. First, estimate white/non-white turnout using ei/rxc. Second, gather overall CVAP numbers. Third, estimate candidate preference by white/non-white using ei/rxc. Then enter values into function.
```
Comment (pssachdeva) [inline R/elect_algebra.R:3]: it'll help the readability if the docstring wraps at 80 characters

**Comment by lorenc5** [comment]
Made all suggested changes and added unit test.

**Pull request merged**